### PR TITLE
Feature/message input toolbar customization

### PR DESCRIPTION
**Pull request by kcoleman731**
* Moved `searchController` property to public API on `ATLConversationListViewController`. 
* Moved `UIImagePickerControllerDelegate` and `UINavigationControllerDelegate` declarations to header of `ATLConversationViewController`.
* Added `leftAccessoryImage`, `rightAccessoryImage` and `displaysRightAccessoryImage` to `ATLMessageInputToolbar`.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Atlas Changelog
 
+## 1.0.4
+
+###Public API Changes
```
Comment (blakewatters) [inline CHANGELOG.md]: Put a space between the headers. That drives me nucking futs

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+* Moved `searchController` property to public API on `ATLConversationListViewController`.
```
Comment (blakewatters) [inline CHANGELOG.md]: Line break before the list starts. Match the style in the file broham.

File: Code/Views/ATLMessageInputToolbar.h [hunk 3]
```diff
@@ -101,6 +101,25 @@ extern NSString *const ATLMessageInputToolbarAccessibilityLabel;
...
+
+/**
+ @abstract The image displayed on right accessory button.
+ @default A `location` icon.
+ */
+@property (nonatomic) UIImage *rightAccessoryImage;
+
+/**
+ @abstract Determines whether or not the right accessory button displays an icon. 
+ @disucssion If NO, the right accessory button will display the text `SEND` at all times.
```
Comment (blakewatters) [inline Code/Views/ATLMessageInputToolbar.h]: Is this really accurate — can’t you switch the text now that you’ve exposed the buttons?

Comment (kcoleman731) [inline Code/Views/ATLMessageInputToolbar.h]: No, we set the text often internally...every time we flip from enabled to disabled....it is necessary for when we switch back and forth between image and text. I can change and make it more flexible if you like.

**Pull request merged**